component/Badge/badge.css

```diff
@@ -0,0 +1,97 @@
+.badge_container {
+    position: relative;
+}
+
+.badge {
+    position: absolute;
+    background-color: var(--primary-color);
+    left: 9.5rem;
+    bottom: 1.1rem;
+    width: 2rem;
+    height: 2rem;
+    border: 3px solid var(--white-color);
+    border-radius: 50%;
+}
+
+.badge_md_size {
+    width: 1.5rem;
+    height: 1.5rem;
+    background-color: var(--danger-alert-color);
+}
+
+.badge_sm_size {
+    height: 1rem;
+    width: 1rem;
+}
+
+.badge_med {
+    position: absolute;
+    left: 7.5rem;
+}
+
+.badge_small {
```

Comment:
follow the same naming convention that you had been following till now. `.badge-md` instead of `.badge-med` and `.badge-sm instead` of `.badge-small`